src/sidebars/favourites/favourites.less

```diff
@@ -26,6 +26,10 @@
       background-color: @highlight-colour;
     }
   }
+
+  .darkens:hover {
+    background: #1F262E;
```

Comment:
Sure. Will pull out a variable and I think there's at least one other colour in that file that's not using one too. Will fix them up too.

The favourites needed an override for the style as the fade version with 0.5 alpha meant the green of the drop-area style showed through for the hover style.